jobs/opensearch/templates/config/opensearch-security/roles_mapping.yml.erb

```diff
@@ -22,10 +22,20 @@ own_index:
   - "*"
   description: "Allow full access to an index named like the username"
 
+index_management_full_access:
+  reserved: false
+  hidden: false
+  users:
+  - "logstash.opensearch.internal"
+  backend_roles:
+  - "logstash.opensearch.internal"
 logstash:
   reserved: false
+  hidden: false
+  users:
+  - "logstash.opensearch.internal"
   backend_roles:
-  - "logstash"
+  - "logstash.opensearch.internal"
```

Comment:
I don't think the change on this line is neceessary

Reply:
It doesn't seem to have done anything good or bad but better to change it back. i still don't understand where this gets its abilities from